### ISSUE TITLE
Does not calculate max-height correctly

### ISSUE THREAD

**Issue by forethoughtde** [nobitagit/squeezebox-vanilla#4]
It cuts off the content in some cases due to miscalculation in max-height
[https://ibb.co/n3Fqoy](https://ibb.co/n3Fqoy)

**Comment by DevAndreyBilchenko** [comment]
same problem

**Comment by SeedyROM** [comment]
Yeah this is super random too, some times it works. other times it doesn't. Sometimes it works half? It's honestly confusing as hell.
